jazz_environment-event-handler/test/COMMIT_TEMPLATE.json

```diff
@@ -0,0 +1,37 @@
+{
+    "Item": {
+      "EVENT_ID": {
+        "S": "a21a6b17-6035-4bcd-90eb-14c7788433a5-007"
+      },
+      "TIMESTAMP": {
+        "S": "2018-03-13T01:18:35:556"
+      },
+      "SERVICE_CONTEXT": {
+        "S": "{\"service_type\":\"nodejs\",\"branch\":\"feature/test\",\"runtime\":\"nodejs4.3\",\"domain\":\"jazztest\",\"iam_role\":\"arn:a:iam::302890901340:role/jazz_platform_services\",\"environment\":\"env_logical_id-dev\",\"region\":\"us-east-1\",\"service_id\":\"5e58d101-2e62-f39c-61ca-83d1e4b5e809\"}"
+      },
+      "USERNAME": {
+        "S": "deepu.sundaresan@ust-global.com"
```

Comment:
obfuscate user name

Reply:
Done,  But we need to check other services as well. There are strings which are base64 encoded and having these kind of information.